webpack.config.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-undef */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.png$/,
+        use: 'file-loader',
+      },
+    ],
+  },
```

Comment:
- [ ] Good job Team 👏 👏 . I faced difficulty while I was trying to run this project locally on my machine because there are some dependencies that were not installed yet IE: Being called in the project but they are not installed. First one is `file-loader` which loads all your image assets into the js bundle. Take a look at the error I was getting 👇 
**In the console**
![Capture2](https://user-images.githubusercontent.com/90222110/189236611-fe72c7fe-e43d-4501-8c8e-1c7cef9ed2e9.PNG)

**In the browser**

![Capture](https://user-images.githubusercontent.com/90222110/189236682-d77649d8-49f7-4008-b4ca-c47ab43b58b2.PNG)

What I would suggest is to install it as a dependency so that It can be available in your project and recorded in the `package.json` 😉